### PR TITLE
 bking nw/create routed, cntrl' meths, view coded with head and half …

### PR DESCRIPTION
**Pull request by akbattman**
…a simple form - awaiting master pull to complete form only.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+    @booking = Booking.new(booking_params)
+    @bookmark.save
+    # redirect_to booking_path(@booking)
+  end
+
+    private
+
+  def booking_params
+    params.require(:booking).permit(:user_id, :chef_id, :start_date, :end_date)
+  end
```
Comment (olya-kolesova) [inline app/controllers/bookings_controller.rb:17]: you don't need user id and chef if :)

File: app/views/bookings/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class='container'>
+  <h4>NEW BOOKING REQUEST</h4>
+  <div class='container'>
+    <%= simple_form_for(@booking) do |f| %>
```
Comment (olya-kolesova) [inline app/views/bookings/new.html.erb:4]: you have two containers i am not sure the second one is useful?

**Review by olya-kolesova — CHANGES_REQUESTED** [review]
Thats Coline :) 👍

File: app/views/bookings/new.html.erb (hunk 2 above)
Comment (akbattman) [reply app/views/bookings/new.html.erb:4]: I always contain the page, and the second container is for the form. 
It is useful when styling.
Feedback received on team concensus. Removing.

**Pull request merged**